src/main/scala/firrtl/stage/TransformManager.scala

```diff
@@ -0,0 +1,27 @@
+// See LICENSE for license details.
+
+package firrtl.stage
+
+import firrtl.{CircuitForm, CircuitState, Transform, UnknownForm}
+import firrtl.options.DependencyManager
+
+class TransformManager(
```

Comment:
Quibble, is this likely to be used by developers, should it have scaladoc

Reply:
Yeah, it should. I'll include documentation in the rebase.